### PR TITLE
feat: Stone Prover config

### PR DESCRIPTION
**Pull request by Jemiiah**
solves issue : #29

### CODE REVIEW

**Comment by Jemiiah** [comment]
alright ser

File: docs/dist/.vocs/icons/arrow-diagonal.svg
Comment (dmirgaleev) [inline docs/dist/.vocs/icons/arrow-diagonal.svg:1]: Please, delete all unnecessary files, only one md file should be in the PR.

File: docs/pages/usage/configuration.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  * `proof_of_work_bits`: This determines the computational difficulty of proof generation, acting as a deterrent against brute force attacks. For example:
+
+    `"proof_of_work_bits": 24`
+
+### 3. Configuration File Format
+The configuration file typically contains parameters that define how the prover and verifier should function in a STARK system, with particular focus on FRI parameters. Here are key sections in the configuration files, as observed from your examples:
+
+Example Structure (`cpu_air_params.json`)
+
+{
```
Comment (dmirgaleev) [inline docs/pages/usage/configuration.md:132]: Please, fix formatting here (```json for json code blocks)

File: docs/pages/usage/configuration.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+  `"log_n_cosets": 4`
+
+* `use_extension_field`: Indicates whether or not to use an extension field for higher security. Example:
+
+  `"use_extension_field": false`
+
+Example Structure (`cpu_air_prover_config.json`)
+
+{
```
Comment (dmirgaleev) [inline docs/pages/usage/configuration.md:162]: Please, fix formatting

File: docs/pages/usage/configuration.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+ii. Step 2: Specify additional fields like `log_n_cosets` and `use_extension_field` based on whether you're using an extension field for added security.
+
+iii.  Step 3: For prover-specific settings (as shown in `cpu_air_prover_config.json`), decide on the task size for constraint polynomial evaluations, whether to store the full LDE, and how many tasks are assigned per segment.
+
+iv. Step 4: Save the configuration file in a `.json` format, adhering to the structure outlined above.
+Example of a custom Configuration File:
+
+**3.2 Example of a Configuration File**
+
+{
```
Comment (dmirgaleev) [inline docs/pages/usage/configuration.md:209]: Please, fix formatting

**Review by dmirgaleev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**